configs/pulpcore/3.15.yaml

```diff
@@ -0,0 +1,130 @@
+---
+:project: pulpcore
+:github_org: pulp
+:nightly: false
+:repos: []
+:releases:
+  :3.15:
+    :redmine_version_id: -1
+:mash_scripts:
+  - pulpcore-3.15-el7
+  - pulpcore-3.15-el8
+:strict_keys: true
+:gpg_key: 6D05E64C
+:tags:
+  - name: pulpcore-3.15-el7
+    based_off: pulpcore-3.14-el7
```

Comment:
Given these configs define the build groups, and external repositories, is the based off even needed?

Reply:
yes, otherwise the deps won't get tagged.

granted, in the case of 3.15, it's not that important, as we need to rebuild everything…